ospatch/system_linux.go

```diff
@@ -116,17 +115,24 @@ func disableAutoUpdates() {
 
 func rebootSystem() error {
 	// Start with systemctl and work down a list of reboot methods.
-	if e := common.Exists(systemctl); e {
+	if e := exists(systemctl); e {
 		return exec.Command(systemctl, "reboot").Start()
 	}
-	if e := common.Exists(reboot); e {
+	if e := exists(reboot); e {
 		return exec.Command(reboot).Run()
 	}
-	if e := common.Exists(shutdown); e {
+	if e := exists(shutdown); e {
 		return exec.Command(shutdown, "-r", "-t", "0").Run()
 	}
 
 	// Fall back to reboot(2) system call
 	syscall.Sync()
 	return syscall.Reboot(syscall.LINUX_REBOOT_CMD_RESTART)
 }
+
+func exists(name string) bool {
```

Comment:
at first i moved this to a common location. i specifically remember you suggesting me to re-write them in the package so that these functions live very close to where thery are used.
and so i just moved them back to individual packaged.
i want to them to be in a common place though; for the sake of re-usability and ofcourse making changes in future becomes easier.

Reply:
i may have miscommunicated, then. i think it's ok if we put this in common

Reply:
i can move this back to a common location so the code is re-usable. however, I had a discussion with Andrew; about the name of the package. We decided to rename the package to util.  Let me know if you have any concerns about it.